### PR TITLE
Robbie/update docs and test plan

### PR DESCRIPTION
**Pull request by rfulton-lf**
updated docs to instruct admins to add a single redirect URI to devconsole and test plan to clarify that login should occur through repository explorer

### CODE REVIEW

File: jekyll_files/docs/admin-documentation/register-app-in-laserfiche.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ parent: Laserfiche SharePoint Integration Administration Guide
    <a href="../assets/images/clientIdRegistered.png"><img src="../assets/images/clientIdRegistered.png"></a>
 1. One way or another, an app with that client ID should now exist. Open the app and switch from the general tab to the authentication tab.
    <a href="../assets/images/redirectUri.png"><img src="../assets/images/redirectUri.png"></a>
-1. Copy the URLs from your three SharePoint Pages with the Laserfiche web part and add them in the developer console as new redirect URIs. Each URI should end in `.aspx`
+1. Copy the URLs from your SharePoint page with the Laserfiche sign in web part, and append the suffix `?autologin` and add it in the developer console as a new redirect URI. The URI should end in `.aspx?autologin`
```
Comment (alexgomezlf) [inline jekyll_files/docs/admin-documentation/register-app-in-laserfiche.md]: Copy the URL from your SharePoint LaserficheSignIn page, append the query parameter `?autologin`  and add it to the developer console as a new redirect URI. The URI should end with `SitePages/LaserficheSignIn.aspx?autologin`.

Also update the screenshot

File: testing/test_plan.md [hunk 2]
```diff
@@ -285,7 +285,7 @@ Prerequisites:
 Steps:
 
 1. Click `Sign in` button
-   - Expected results: You are led through the OAuth flow, you return to repository explorer page, and button says `Sign Out`
+   - Expected results: You are led through the OAuth flow via a popup window, you return to repository explorer page, and button says `Sign Out`.
```
Comment (alexgomezlf) [inline testing/test_plan.md:294]: There are also changes regarding number validation so when you test with 'Actual Number' and with the Create folder dialog

File: jekyll_files/docs/admin-documentation/register-app-in-laserfiche.md [hunk 3]
```diff
@@ -21,7 +21,7 @@ parent: Laserfiche SharePoint Integration Administration Guide
    <a href="../assets/images/clientIdRegistered.png"><img src="../assets/images/clientIdRegistered.png"></a>
 1. One way or another, an app with that client ID should now exist. Open the app and switch from the general tab to the authentication tab.
    <a href="../assets/images/redirectUri.png"><img src="../assets/images/redirectUri.png"></a>
-1. Copy the URLs from your three SharePoint Pages with the Laserfiche web part and add them in the developer console as new redirect URIs. Each URI should end in `.aspx`
+1. Copy the URL from your SharePoint LaserficheSignIn page, append the query paramter `?autologin` and add it to the developer console as a new redirect URI. The URI should end with `SitePages/LaserficheSignIn.aspx?autologin`.
```
Comment (alexgomezlf) [inline jekyll_files/docs/admin-documentation/register-app-in-laserfiche.md]: parameter

File: jekyll_files/docs/assets/images/urlExample.png
Comment (alexgomezlf) [inline jekyll_files/docs/assets/images/urlExample.png:1]: Did you switch it pack to an image of the dev console?

File: testing/test_plan.md [hunk 4]
```diff
@@ -368,7 +368,7 @@ Test delete after save action
     - Expected results: Dialog remains open, error specifies to provide a folder name
 1. Close dialog
 1. Use create folder button
-1. Use name with invalid characters (e.g. `~!@#$%^&*()` )
+1. Use name with the invalid character `\`.
```
Comment (alexgomezlf) [inline testing/test_plan.md:377]: Test saving one with other characters and it does save

**Review by alexgomezlf — DISMISSED** [review]

File: jekyll_files/docs/assets/images/urlExample.png
Comment (rfulton-lf) [reply jekyll_files/docs/assets/images/urlExample.png:1]: I did, but on second thought it makes more sense to screenshot the sign in page. Thanks.

File: testing/test_plan.md [hunk 5]
```diff
@@ -368,9 +368,14 @@ Test delete after save action
...
 1. Attempt to create folder
     - Expected results: Dialog remains open, error  specifies to provide a valid folder name
+1. Use create folder button.
+1. Use name with alphanumeric and with someone of the valid special characters `!@#$%^&*()`.
```
Comment (alexgomezlf) [inline testing/test_plan.md]: "with some of"

Comment (rfulton-lf) [reply testing/test_plan.md]: goodness gracious. thanks

**Review by alexgomezlf — APPROVED** [review]

**Pull request merged**